<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="color_format" value="yuv422p" />
    <!-- <param name="pixel_format" value="mjpeg" /> -->
    <param name="focus_auto" value="false" />
    <param name="camera_frame_id" value="usb_cam" />
    </node>
    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/usb_cam/image_raw"/>
        <param name="autosize" value="true" />
    </node> 
        <!-- <node name="arrow_detector" pkg="detection" type="arr_detect.py" output="screen" />   -->  <!--this is the node for launching the arrow detector with contour-->
        <!-- <node name="arrow_detector_svm" pkg="detection" type="arr_detect_svm.py" output="screen" /> --><!--this is the node for launching the arrow detector with Support Vector Machine-->
        <node name="realtime_arrow_detection_node" pkg="detection" type="arr_detect_cnn.py" output="screen" /> <!--this is the node for launching the arrow detector with EfficientNetB CNN Deep Learning Model-->
</launch>
